packages/openactive-integration-tests/test/flows/book-and-cancel/book-and-customer-cancel-failure/book-shared.js

```diff
@@ -0,0 +1,277 @@
+const chakram = require("chakram");
+const expect = chakram.expect;
+const pMemoize = require("p-memoize");
+const RequestHelper = require("../../../helpers/request-helper");
+const Logger = require("../../../helpers/logger");
+const sharedValidationTests = require("../../../shared-behaviours/validation");
+
+function performTests(dataItem) {
+  const { event: testEvent, price, name: eventName } = dataItem;
+
+  var opportunityId;
+  var offerId;
+  var sellerId;
+  var uuid;
+  var totalPaymentDue;
+  var orderItemId;
+
+  var c1Response;
+  var c2Response;
+  var bResponse;
+  var uResponse;
+  var getOrderPromise;
+  var ordersFeedUpdate;
```

Comment:
Should these use `let`, for consistency?

Reply:
Mostly kept it for consistency with Nicks original var declarations, but yeah, could be time to change those over.

Reply:
I see - in that case ✅